terra2/assetlist.json

```diff
@@ -1759,6 +1690,109 @@
       "name": "Eris Protocol LUNA/USDC ampLP",
       "display": "amplp",
       "symbol": "LUNA-USDC-ampLP"
+    },
+    {
+      "description": "EURe is a Euro-backed stablecoin issued by Monerium on Noble.",
+      "denom_units": [
+        {
+          "denom": "ibc/8D52B251B447B7160421ACFBD50F6B0ABE5F98D2C404B03701130F12044439A1",
+          "exponent": 0,
+          "aliases": [
+            "ueure"
+          ]
+        },
+        {
+          "denom": "eure",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/8D52B251B447B7160421ACFBD50F6B0ABE5F98D2C404B03701130F12044439A1",
```

Comment:
Error: Trace of terra2, ibc/8D52B251B447B7160421ACFBD50F6B0ABE5F98D2C404B03701130F12044439A1 makes invalid reference to noble, eure.